modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp

```diff
@@ -2114,6 +2114,51 @@ PERF_TEST_P_(ResizePerfTest, TestPerformance)
     SANITY_CHECK_NOTHING();
 }
 
+PERF_TEST_P_(StackOverflowPerfTest, TestPerformance)
```

Comment:
Please rename this test to reflect what it actually tests, or add a verbose comment with a link to StackOverflow discussion and a brief description

Reply:
Done.